modules/ref_fb_module/src/scaling_fb_impl.cpp

```diff
@@ -110,23 +110,21 @@ void ScalingFbImpl::processSignalDescriptorChanged(const DataDescriptorPtr& inpu
 
 void ScalingFbImpl::configure()
 {
+    setComponentStatus(ComponentStatus::Ok);
```

Comment:
Why do you set to OK at the start of method. Simply call this as last step in try..catch:

try
{
  ..
  ..
  ..
  SetStatusToOK
}
catch
{
  SetStatusToWarn/Err
}